scenario_editor/src/components/Welcome.js

```diff
@@ -1,60 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
im confused by the components folder in general. It seems arbitrary what is in there and what isn't. Also inside of components you have an images folder, that should definitely be in public not components